examples/ai-core/package.json

```diff
@@ -24,15 +24,18 @@
     "@ai-sdk/replicate": "0.1.7",
     "@ai-sdk/togetherai": "0.1.9",
     "@ai-sdk/xai": "1.1.8",
-    "@opentelemetry/sdk-node": "0.54.2",
+    "@google/generative-ai": "0.21.0",
     "@opentelemetry/auto-instrumentations-node": "0.54.0",
+    "@opentelemetry/sdk-node": "0.54.2",
     "@opentelemetry/sdk-trace-node": "1.28.0",
     "ai": "4.1.21",
     "dotenv": "16.4.5",
+    "image-type": "^5.2.0",
     "mathjs": "14.0.0",
+    "sharp": "^0.33.5",
+    "terminal-image": "^2.0.0",
```

Comment:
are all 3 new dependencies needed?

Reply:
Keeping them for now, we can shift around in time. 

We could do work to force `png` format for replicate (the `webp` use case) and then we wouldn't need `sharp`, but we intend to add support to specify alternate formats, and then we might need it again.

Per other comment, we can switch to our own image type detection separately.